src/modules/achievementMethods.ts

```diff
@@ -50,6 +60,26 @@ export const achievementMethods = {
             return user;
         }
 
+        if (achievement.bonusForAchievementRule && config.techAdminId) {
```

Comment:
What is techAdminId?

Reply:
technical account to give bonuses and achievements automatically